_data/retail.yml

```
@@ -337,12 +337,12 @@ websites:
       software: Yes
       doc: https://help.shopify.com/manual/your-account/account-security/two-step-authentication
 
-    - name: Sierra Trading Post
-      url: https://www.sierratradingpost.com
-      img: sierratradingpost.png
+    - name: Sierra
+      url: https://www.sierra.com
+      img: sierra.png
       tfa: No
       twitter: Sierratp
```

The original twitter handle looks like a fun spam account 😆 
```suggestion
      twitter: sierraofficial
```